x-pack/platform/plugins/shared/actions/server/lib/axios_utils.ts

```diff
@@ -13,12 +13,13 @@ import type {
   AxiosRequestConfig,
   AxiosHeaderValue,
 } from 'axios';
-import { AxiosHeaders } from 'axios';
+import { AxiosHeaders, isAxiosError } from 'axios';
 import type { Logger } from '@kbn/core/server';
 import { getCustomAgents } from './get_custom_agents';
 import type { ActionsConfigurationUtilities } from '../actions_config';
-import type { ConnectorUsageCollector, SSLSettings } from '../types';
+import type { ConnectorUsageCollector, SSLSettings, ErrorCategorizationOverrides } from '../types';
 import { combineHeadersWithBasicAuthHeader } from './get_basic_auth_header';
+import { handleActionHttpUserErrors } from './categorize_http_errors';
 
 export const request = async <T = unknown>({
```

Comment:
This change makes sense to be here because it is what the `sub_actions_connector` uses. That means all connectors will be affected, as they should.

However, I took a look at where else this `request` method was being used, and there are a couple more places:
- `request_oauth_token`
- `request_oauth_jwt_token`
- `request_oauth_client_credentials_token`

These are not necessarily wrong. Since these are tokens fetched using connector data, an error with these codes might still be a user error.

However, could we please double-check that that is the case? I'd not like to run into a scenario where some of these errors are wrongly characterized and end up hiding some problem we have in our connector auth implementation.

Reply:
@cnasikas shall I disable the categorization in these functions? They're ultimately used by `ActionsClient.getOAuthAccessToken()` and ServiceNow's `createServiceWrapper()`